src/docs/truffle/reference/configuration.md

```diff
@@ -166,27 +166,123 @@ mocha: {
 }
 ```
 
-## Solidity compiler configuration
+## Compiler configuration
 
-Solidity compiler settings. Supports optimizer settings for `solc`.
+In the `compilers` object you can specify settings related to the compilers used by Truffle.
 
 ### solc
 
-Configuration options to pass to the Solidity compiler.
+Solidity compiler settings. Supports optimizer settings for `solc`.
 
-Example:
+You may specify...
++ any solc-js version listed at [solc-bin](http://solc-bin.ethereum.org/bin/list.json). Specify the one you want and Truffle will get it for you.
++ a natively compiled solc binary (you'll need to install this yourself, links to help below).
++ a dockerized solc from one of images published [here](https://hub.docker.com/r/ethereum/solc/tags/). (You'll also need to pull down the docker image yourself but it's really easy.)
++ a path to a locally available solc
+
+Truffle config example:
 
 ```javascript
-solc: {
-  optimizer: {
-    enabled: true,
-    runs: 200
+module.exports = {
+  compilers: {
+    solc: {
+      version: <string>,     // A version or constraint - Ex. "^0.5.0"
+      optimizer: {
```

Comment:
This should be inside `settings`

Reply:
Ah man I can't believe I made that mistake after all the hub bub